dotnet/src/IntegrationTests/PromptTests.cs

```diff
@@ -57,7 +57,7 @@ public async Task GenerateStoryTestAsync(string resourceName, bool isHandlebars)
             });
 
         // Assert
-        Assert.Contains("Dog", actual.GetValue<string>(), StringComparison.OrdinalIgnoreCase);
+        Assert.True(actual.GetValue<string>()?.Length > 0);
```

Comment:
This verification isn't reliable e.g. the story my reference the dog breed as shown below

```
[xUnit.net 00:00:02.03]         Function completed. Duration: 1.2524791s
  Failed SemanticKernel.IntegrationTests.PromptTests.GenerateStoryTestAsync(resourceName: "SemanticKernel.IntegrationTests.prompts.GenerateSt"···, isHandlebars: True) [1 s]
  Error Message:
   Assert.Contains() Failure: Sub-string not found
String:    "\n\nRex, a golden retriever, eagerly wagged"···
Not found: "Dog"
```